fragment_internal_test.go

```diff
@@ -2334,12 +2359,12 @@ func getUpdataInto(f func(row, col uint64) bool, numRows, numCols uint64, seed i
 
 // getZipfRowsSliceStandard is the same as getZipfRowsSliceRoaring, but returns
 // row and column ids instead of a byte slice containing roaring bitmap data.
-func getZipfRowsSliceStandard(numRows uint64, seed int64) (rowIDs, columnIDs []uint64) {
+func getZipfRowsSliceStandard(numRows uint64, seed int64, startFrom, endAt uint64) (rowIDs, columnIDs []uint64) {
```

Comment:
these should use the same name, whether it's startCol or startFrom, I think.

Reply:
yeah, across 4 args I used 3 different conventions... maybe not the best